src/spherexlander/themes/spherex/site/index.html.jinja

```diff
@@ -16,18 +16,69 @@
 {% block info_header %}
 <header class="lander-info-header">
   <img src="spherex-logo.png" width="120px" />
-  <h1 class="lander-h1 lander-info-header__title">{{ metadata.title }}</h1>
-  {% if metadata.pipeline_level %}
-  <p class="primary-metadata">{{ metadata.pipeline_level }} pipeline
-    {% if metadata.diagram_index %} (diagram #{{ metadata.diagram_index }}){% endif %}{%- if metadata.difficulty %}, {{ metadata.difficulty.lower() }} difficulty{% endif %}
+
+  {% if metadata.document_handle_prefix %}
+  <p class="spherex-document-category">
+    {% if metadata.document_handle_prefix == "SSDC-RQ" %}
+      SSDC Requirements Document
+    {% elif metadata.document_handle_prefix == "SSDC-PM" %}
+      SSDC Project Management Document
+    {% elif metadata.document_handle_prefix == "SSDC-MS" %}
+      SSDC Pipeline Module Specification
+    {% elif metadata.document_handle_prefix == "SSDC-DP" %}
+      SSDC Data Product Specification
+    {% elif metadata.document_handle_prefix == "SSDC-TN" %}
+      SSDC Technical Note Specification
+    {% elif metadata.document_handle_prefix == "SSDC-IF" %}
+      SSDC Interface Specification
+    {% elif metadata.document_handle_prefix == "SSDC-TR" %}
+      SSDC Test/Technical Report
+    {% elif metadata.document_handle_prefix == "SSDC-RV" %}
+      SSDC Review Material
+    {% elif metadata.document_handle_prefix == "SSDC-OP" %}
+      SSDC Operations Procedure
+    {% elif metadata.document_handle_prefix == "SSDC-OP" %}
+      SSDC Operations Procedure
+    {% endif %}
   </p>
   {% endif %}
-  {% if metadata.version %}
-  <p class="version-metadata">
-    Version {{ metadata.version }}{%- if metadata.status %}, status: {{ metadata.status }} {% endif %}
-  </p>
+
+  <h1 class="lander-h1 lander-info-header__title">{{ metadata.title }}</h1>
+
+  {% if metadata.identifier %}
+  <p class="spherex-handle">{{ metadata.identifier }}</p>
   {% endif %}
 
+  <div class="spherex-metadata-badges">
```

Comment:
If this language allows a comment, a comment here that says "The colors below are taken from Brendan Crill's pipeline overview diagram." would be a good idea.